### PR TITLE
Queue fix

### PR DESCRIPTION
**Pull request by katyhuff**
When tracking down build issue, I ran accross a known issue in which multiprocessing.Queue.qsize() is NotImplemented on Mac.

I was getting one of these:

```
ERROR:procbuild:Exception on /build/0 [GET]
Traceback (most recent call last):
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/huff/procbuild/procbuild/server.py", line 112, in build
    return real_build(*args, **kwarg)
  File "/Users/huff/procbuild/procbuild/server.py", line 96, in real_build
    if paper_queue.qsize() >= 50:
  File "/Users/huff/anaconda3/envs/py27/lib/python2.7/multiprocessing/queues.py", line 143, in qsize
    return self._maxsize - self._sem._semlock._get_value()
NotImplementedError
```

So I went with a solution similar to this one: https://sourceforge.net/p/blockit/support-requests/1/

Also, I slipped in an unrelated change to the readme to support PR in scipy_proceedings:  https://github.com/scipy-conference/scipy_proceedings/pull/190

### CODE REVIEW

File: build_papers.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Schedule some or all papers for build
 
-from procbuild import papers, paper_queue
+from procbuild import papers, paper_queue
```
Comment (stefanv) [inline build_papers.py]: :)

File: build_papers.py [hunk 2]
```diff
@@ -12,8 +12,9 @@
 
 for p in to_build:
     print "Placing %s in the build queue." % p
-    paper_queue.put(int(p))
+    paper_queue[0].put(int(p))
+    paper_queue[1] += int(p)
```
Comment (stefanv) [inline build_papers.py]: Should this be `+= 1` perhaps?

File: procbuild/server.py [hunk 3]
```diff
@@ -74,12 +74,12 @@ def _process_queue(queue):
             log("Sentinel found in queue. Ending queue monitor.")
             done = True
         else:
-            log("Queue yielded paper #%d. Left: %d" % (nr, queue.qsize()))
+            log("Queue yielded paper #%d." % nr)
```
Comment (stefanv) [inline procbuild/server.py]: Not accessible from `paper_queue`?

File: runserver.py [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 # -- SERVER CONFIGURATION -- (can be overridden from shell)
 
-config = (('MASTER_BRANCH', '2015'),
-          ('ALLOW_MANUAL_BUILD_TRIGGER', '0'))
+config = (('MASTER_BRANCH', '2016'),
+          ('ALLOW_MANUAL_BUILD_TRIGGER', '1'))
```
Comment (stefanv) [inline runserver.py]: This should probably stay zero by default.

File: procbuild/server.py (hunk 3 above)
Comment (katyhuff) [inline procbuild/server.py]: paper_queue might not be the queue passed to the function.

Comment (stefanv) [inline procbuild/server.py]: Got it​

File: runserver.py (hunk 4 above)
Comment (katyhuff) [inline runserver.py]: any particular reason? People used it really often last year to rebuild their papers. Anyway, it's set to 1 in the __init__ file in procbuild. It seemed more natural for them to agree.

File: build_papers.py (hunk 2 above)
Comment (katyhuff) [inline build_papers.py]: not if p > 1 ....

Comment (katyhuff) [inline build_papers.py]: oh wait, I got you. you're right.

**Comment by stefanv** [comment]
This fix never decreases the queue size, so after 50 papers have been built, the builder gets stuck.

**Pull request merged**